h2o-parsers/h2o-orc-parser/src/test/java/water/parser/ParseTestORCCSV.java

```diff
@@ -20,61 +23,65 @@
  */
 public class ParseTestORCCSV extends TestUtil {
 
-    private String[] csvFiles = {"smalldata/parser/orc/orc2csv/TestOrcFile.testDate1900.csv",
-            "smalldata/parser/orc/orc2csv/TestOrcFile.testDate2038.csv",
-            "smalldata/parser/orc/orc2csv/orc_split_elim.csv", "smalldata/parser/csv2orc/prostate_NA.csv",
-            "smalldata/iris/iris.csv", "smalldata/jira/hexdev_29.csv"};
+    private String[] csvFiles = {"smalldata/parser/orc/orc2csv/testTimeStamp.csv",
+            "smalldata/parser/orc/orc2csv/testDate1900.csv",
+            "smalldata/parser/orc/orc2csv/testDate2038.csv"};
 
-    private String[] orcFiles = {"smalldata/parser/orc/TestOrcFile.testDate1900.orc",
-            "smalldata/parser/orc/TestOrcFile.testDate2038.orc", "smalldata/parser/orc/orc_split_elim.orc",
-            "smalldata/parser/orc/prostate_NA.orc", "smalldata/parser/orc/iris.orc",
-            "smalldata/parser/orc/hexdev_29.orc"};
+    private String[] orcFiles = {"smalldata/parser/orc/testTimeStamp.orc",
+            "smalldata/parser/orc/TestOrcFile.testDate1900.orc",
+            "smalldata/parser/orc/TestOrcFile.testDate2038.orc"};
 
-    private Boolean[] forceColumnTypes = {false, false, false, true, true, true};
+    private Boolean[] forceColumnTypes = {false, false, false, false, true, true, true};
```

Comment:
A better solution would be to have a Set<Integer> containing just 2,3,4, and checking forceColumnTypes.contains(n)